docs/en/logs/using.asciidoc

```diff
@@ -0,0 +1,97 @@
+==== Use the power of search
+
+To perform ad hoc searches for specific text, use the search bar.
+You can also create structured queries using {kibana-ref}/kuery-query.html[Kibana Query Language].
+For example, enter `host.hostname : "host1"` to see only the information for `host1`.
+// ++ this isn't quite the same as the corresponding metrics description now.
+
+[float]
+[[logs-configure-source]]
+==== Configure the data to use for your logs
+Are you using a custom index pattern to store the log entries?
+Do you want to limit the entries shown or change the fields displayed in the columns?
+If so, <<xpack-logs-configuring, configure the logs source data>> to change the index pattern and other settings.
+
+[float]
+[[logs-time]]
+==== Specify the time and date
+
+Click image:images/time-filter-calendar.png[time filter calendar], then choose the time range for the logs.
+
+Log entries for the specified time appear in the middle of the page. To quickly jump to a nearby point in time, click the minimap timeline to the right.
+// ++ what's this thing called? It's minimap in the UI. Would timeline be better?
+
+[float]
+[[logs-customize]]
+==== Customize your view
+Click *Customize* to customize the view.
+Here, you can choose whether to wrap long lines and select your preferred text size.
+
+[float]
+==== Configuring the data to use for your logs
+
+If your logs have custom index patterns, use non-default field settings, or contain parsed fields which you want to expose as individual columns, you can <<xpack-logs-configuring, override the default configuration settings>>.
+
+[[logs-stream]]
+=== Stream or pause logs
+Click *Stream live* to start streaming live log data or click *Stop streaming* to focus on historical data.
+
+When you are viewing historical data, you can scroll back through the entries as far as there is data available.
+
+When you are streaming live data, the most recent log appears at the bottom of the page.
+In live streaming mode, you are not able to choose a different time in the time selector or use the minimap timeline.
+To do either of these things, you need to stop live streaming first.
+// ++ Not sure whether this is correct or not. And what about just scrolling through the display?
+// ++ There may be a bug here, (I managed to get future logs) see https://github.com/elastic/kibana/issues/43361
+
+[float]
+[[logs-highlight]]
+=== Highlight a phrase in the logs stream
+To highlight a word or phrase in the logs stream, click *Highlights* and enter your search phrase.
+// ++ Is search case sensitive?
+// ++ Can you search for multiple phrases together, if so, what's the separator?
+// ++ What about special characters? For example, I notice that when searching for "Mozilla/4.0" which appears as written in my logs, "Mozilla" is highlighted, as is "4.0" but "/" isn't. The string "-" (which appears in the logs as written, quotes and all, isn't found at all. Any significance?
+
+[[logs-event-inspector]]
+=== Inspect log events
```

Comment:
It looks like a lot of the content on this page isn't floated. Did you mean to do that? I like it, especially with the task-based headings. Just want to confirm it's what you had in mind.

<img width="363" alt="Screen Shot 2020-06-09 at 11 23 21 AM" src="https://user-images.githubusercontent.com/5618806/84185315-ac643300-aa43-11ea-9a57-dcc03eefad8c.png">

Reply:
Yeah, I wanted to create smaller task-based content along with the headings.

Reply:
Perfect

Reply:
I'll revisit the Uptime monitoring guide and do likewise.